### ISSUE TITLE
How to monitor training with tensorboard without keras aplication

### ISSUE THREAD

**Issue by evanmartua34** [google/automl#636]
Hi I actually a beginner in tensorflow object detection API, I wonder how to train efficientdet without keras application while using tensorboard to monitor the performance (mAP and loss) ? if it can be done. Thank you

**Comment by kcrumb** [comment]
If I understand you correctly then you need to call `main.py --mode=...` and start tensorboard in the `model_dir` directory. This should use the estimator API. Look under [Finetune](https://github.com/google/automl/tree/master/efficientdet#8-finetune-on-pascal-voc-2012-with-detector-coco-ckpt) for an example.

**Comment by evanmartua34** [comment]
I see I will try to look further, thank you
